news/_posts/core-weekly/2020-11-09-coreweekly-45-2020.md

```diff
@@ -0,0 +1,262 @@
+
+
+### Issues Bot
+* [#10](https://github.com/PrestaShop/issuebot/pull/10): Remove Opening Message, by [@Progi1984](https://github.com/Progi1984)
+
+
+### Docker images
+* [#237](https://github.com/PrestaShop/docker/pull/237): Update license href in README, by [@matks](https://github.com/matks)
+
+
+### PrestaShop Specifications
+* [#166](https://github.com/PrestaShop/prestashop-specs/pull/166): [multistore] - Add UAT for multistore, by [@marionf](https://github.com/marionf)
+* [#165](https://github.com/PrestaShop/prestashop-specs/pull/165): Add SVG as accepted logo format, by [@LouiseBonnard](https://github.com/LouiseBonnard)
+* [#163](https://github.com/PrestaShop/prestashop-specs/pull/163): Specify the Modules Manager > Updates page, by [@LouiseBonnard](https://github.com/LouiseBonnard)
+* [#162](https://github.com/PrestaShop/prestashop-specs/pull/162): Update Add new Order.md. Thank you [@MatShir](https://github.com/MatShir)
+* [#161](https://github.com/PrestaShop/prestashop-specs/pull/161): Specify the default language for order documents, by [@LouiseBonnard](https://github.com/LouiseBonnard)
+* [#157](https://github.com/PrestaShop/prestashop-specs/pull/157): Create specs's page database. Thank you [@MatShir](https://github.com/MatShir)
+* [#154](https://github.com/PrestaShop/prestashop-specs/pull/154): Add the overrides list to the Information specs, by [@LouiseBonnard](https://github.com/LouiseBonnard)
+* [#153](https://github.com/PrestaShop/prestashop-specs/pull/153): Update the product settings specifications, by [@LouiseBonnard](https://github.com/LouiseBonnard)
+* [#152](https://github.com/PrestaShop/prestashop-specs/pull/152): Create order settings specs, by [@marionf](https://github.com/marionf)
+* [#151](https://github.com/PrestaShop/prestashop-specs/pull/151): Add the free gift behavior in FO, by [@LouiseBonnard](https://github.com/LouiseBonnard)
+* [#135](https://github.com/PrestaShop/prestashop-specs/pull/135): Add specs for BO menu tab, by [@marionf](https://github.com/marionf)
+
+
+### The PrestaShop open source project
+* [#49](https://github.com/PrestaShop/open-source/pull/49): Add qa-core Slack channel, by [@matks](https://github.com/matks)
+
+
+### Wishlist block module
+* [#89](https://github.com/PrestaShop/blockwishlist/pull/89): Get cover image for specific combination. Thank you [@kpodemski](https://github.com/kpodemski)
+
+
+### Email Alerts module
+* [#65](https://github.com/PrestaShop/ps_emailalerts/pull/65): Fix cast error when removing an Alert. Thank you [@hherreros93](https://github.com/hherreros93)
+* [#57](https://github.com/PrestaShop/ps_emailalerts/pull/57): Added new translator for email subjects. Thank you [@L3RAZ](https://github.com/L3RAZ)
+
+
+### Visits and Visitors statistics module
+* [#11](https://github.com/PrestaShop/statsvisits/pull/11): Release 2.0.3, by [@Progi1984](https://github.com/Progi1984)
+
+## Logo and description changes
```

Comment:
this one is empty

Reply:
No, it's the title with two `#` then you see all modules with three `#`.

Reply:
My bad. I didn't understand that was a specific section for this weekly.